src/taco-remote-lib/ios/iosBuild.ts

```diff
@@ -79,11 +79,18 @@ class IOSBuilder extends Builder {
     protected collectVersionInfo(): Q.Promise<any> {
         var self: IOSBuilder = this;
 
+        // Try to get the cordova-ios version
+        try {
+            self.cordovaIosVersion= require(path.join(this.currentBuild.appDir, "platforms", "ios", "cordova", "version")).version;
+        } catch (e) {
+            Logger.log(resources.getString("CordovaIosVersionNumberFetchFailed"));
+        }
+
         // Get the version of cordova-ios and XCode currently installed
         let execDeferred: Q.Deferred<string> = Q.defer<string>();
-        child_process.exec("xcodebuild -version && cordova platforms version ios", { cwd: this.currentBuild.appDir } , function (error, stdout, stderr) {
+        child_process.exec("xcodebuild -version", { cwd: this.currentBuild.appDir } , function (error, stdout, stderr) {
             if (error) {
-                execDeferred.reject(new Error(resources.getString("XCode83GetIosPlatformVersionWarning")));
```

Comment:
This string was already missing and doesn't need to be removed from the string resources, right?